### ISSUE TITLE
HTML output cleanup

### ISSUE THREAD

**Issue by iimpulse** [TheJacksonLaboratory/LIRICAL#467]
Enhance the output html with css.

**Comment by pnrobinson** [comment]
@iimpulse Note that the CSS is inline. I am right now about to check in a big change to the templat. Please let me know before you start working!

**Comment by pnrobinson** [comment]
@jmcmurry @iimpulse  I have taken up Julie's great suggestion and implemented a new table with sparklines to represent the phenotype profiles. It is starting to look decent but let's try to polish this week. There are still a few HTML/CSS issues that are suboptimal.

![image](https://user-images.githubusercontent.com/4943420/68710281-3a874200-0565-11ea-80cd-248fa4f9afe1.png)

**Comment by jmcmurry** [comment]
Great! 
- The text seems all at different sizes; would be great to make consistent
- Wrap header text in composite LR (log)
- Might be good to make the whole row clickable (rather than just the disease) so that is clear there's info to drill down to no matter what they're looking at specifically. Even better would be to somehow highlight the whole row on hover to make its clickableness that much more apparent to user. Hover stuff not terribly mobile friendly; not sure how much it matters here.
- Unless you're retaining the same exact phenotype at the same vertical position in the sparklines (a fine approach), it would be better to sort the phenotypes from most to least contributing. That gives user a better overall feel for the quality of the matches.
Keep up the great work :)

**Comment by pnrobinson** [comment]
@jmcmurry  -- I do not see this app being used on a mobile at all! I have adjusted the size of the text in the sparkline table, but it seems best to keep the disease name in smaller font because of the typical length of the names. It is hard to do this exactly because the SVG and the HTML are using different ways of calculating font size.
The last point is the same as Justin's but I think that users will like to see the phenotypes ordered in the same way for each candidate. This allows you to see at a glance if one HPO conributes to multiple diseases or not -- we would lose this is we sorted the sparklines for each disease.
